### DISCUSSION TITLE
Docker Image

### DISCUSSION THREAD

**Discussion by Idan-fishman-old** [SAP/PyRFC#267] [Ideas]
Hi 👋 
In our organization we are developing many projects for BASIS automations using **pyrfc**. therefore I have developed a Docker image that contains Pyrfc 2.50 with NW RFC SDK PL9, python3.10.2 in a small size of only **170MB**.
I would like to contribute to this repo with my Docker Image :)
@bsrdjan

**Comment by amarvin** [comment]
I believe SAP has decided that SAP NW RFC SDK (a dependency of `pyrfc`) is proprietary, and only available for download by people with SAP customer S-user accounts. So I wouldn't try to publish such a Docker image. I wish it was all open source though!

**Reply by Idan-fishman-old** [reply]
I also know that but, we can publish a docker file and comment the code where we are actually copying SAP NW RFC SDK to the image. 
add some instructions as comments for people with S-user accounts so they will be able to start using the image with the files they downloaded using their S-users.

**Reply by bsrdjan** [reply]
@Dan-Fishman, the docker image without RFC SDK is welcome, can pe published.

**Reply by Idan-fishman-old** [reply]
Great,I will push it to dockerhub.

**Reply by bsrdjan** [reply]
@Idan-fishman-old you can share the reference to image so that users can find it also via this thread
